android-sdk/src/main/java/ai/evolv/android_sdk/helper/UtilityHelper.java

```diff
@@ -13,4 +21,26 @@
             return null;
         }
     }
+
+    public JsonElement setKeyToValue(String key, JsonElement value, JsonElement map){
```

Comment:
Me ntioned unit test suite before, but something like this really needs tests

Reply:
I will make unit tests for it 
tnx